airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/manifest.yaml

```diff
@@ -0,0 +1,924 @@
+            type:
+              - "null"
+              - string
+          location:
+            type:
+              - "null"
+              - string
+          phones:
+            type:
+              - "null"
+              - array
+            items:
+              type: object
+          emails:
+            type:
+              - "null"
+              - array
+            items:
+              type: string
+          links:
+            type:
+              - "null"
+              - array
+            items:
+              type: string
+          archived:
+            type:
+              - "null"
+              - object
+          tags:
+            type:
+              - "null"
+              - array
+            items:
+              type: string
+          sources:
+            type:
+              - "null"
+              - array
+            items:
+              type: string
+          stageChanges:
+            type:
+              - "null"
+              - array
+            items:
+              type: object
+          origin:
+            type:
+              - "null"
+              - string
+          sourcedBy:
+            type:
+              - "null"
+              - string
+          owner:
+            type:
+              - "null"
+              - string
+          followers:
+            type:
+              - "null"
+              - array
+            items:
+              type: string
+          applications:
+            type:
+              - "null"
+              - array
+            items:
+              type: string
+          createdAt:
+            type:
+              - "null"
+              - integer
+          updatedAt:
+            type:
+              - "null"
+              - integer
+          lastInteractionAt:
+            type:
+              - "null"
+              - integer
+          lastAdvancedAt:
+            type:
+              - "null"
+              - integer
+          snoozedUntil:
+            type:
+              - "null"
+              - integer
+          urls:
+            type:
+              - "null"
+              - object
+          resume:
+            type:
+              - "null"
+              - string
+          dataProtection:
+            type:
+              - "null"
+              - object
+          isAnonymized:
+            type:
+              - "null"
+              - boolean
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: updatedAt
+      cursor_datetime_formats:
+        - "%ms"
+      datetime_format: "%ms"
+      start_datetime:
+        type: MinMaxDatetime
+        datetime: "{{ config['start_date'] }}"
+        datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+      start_time_option:
+        type: RequestOption
+        field_name: updated_at_start
+        inject_into: request_parameter
+      end_time_option:
+        inject_into: request_parameter
+        type: RequestOption
+        field_name: updated_at_end
```

Comment:
Do we need a date range end here? Now, the connector is sending the request for every month instead of one request to get all the data

Reply:
Yes, we need the date range end here, I tried removing it and running the stream, it is returning duplicate records. I'll change the granularity to `PT0.000001S` so that we don't miss any data.